.gitignore

```diff
@@ -1 +1,2 @@
 /target
+.vscode/launch.json
```

Comment:
It's an endless debate but I prefer tool specific configuration to be outside of the project. For this I use a custom git ignore in my home